### DISCUSSION TITLE
Are there incompatibilities between dirvish and dired-x?

### DISCUSSION THREAD

**Discussion by teehemkay** [alexluigit/dirvish#76] [Q&A] [answered]
I'm asking because of this https://github.com/doomemacs/doomemacs/commit/1e6d5c3f65512fa2335de5ecd9e09718a1a02ae6#r78281866

**Comment by alexluigit** [comment, marked as answer]
> Are there incompatibilities between dirvish and dired-x?

Not that I know of. Dirvish should work fine with builtin packages.
